presto-docs/src/main/sphinx/connector/base-arrow-flight.rst

```diff
@@ -24,6 +24,9 @@ To utilize the base-arrow-module, you need to implement certain abstract methods
 * ``ArrowPlugin.java``
   Register your connector name by extending the ArrowPlugin class.
 
+* ``ArrowFlightRequest``
+  The ``getCommand`` method in the ``ArrowFlightRequest`` interface is responsible for serializing a flight request object into a byte array.
```

Comment:
Can reword like "The getCommand method in ........ should return a byte array for the flight request.